WcaOnRails/spec/factories/persons.rb

```diff
@@ -3,9 +3,11 @@
   factory :person do
     wca_id do
       mid = ('A'..'Z').to_a.sample(4).join
-      "2016#{mid}01".tap do |id|
-        id.next! while Person.exists?(wca_id: id)
+      id = "2016#{mid}01"
+      while Person.exists?(wca_id: id)
+        id = id.next
       end
```

Comment:
Rubocop dislikes this ([=)](https://github.com/bbatsov/ruby-style-guide#while-as-a-modifier)).